source/sass/global/variables.scss

```
@@ -0,0 +1,17 @@
+// Colors global
+
+$basic-white: #ffffff;
+$basic-black: #000000;
+$basic-green: #68b738;
+
+// Colors special
+
+$background-green: rgba(104, 183, 56, 0.85);
```

С препроцессорами так можно не делать и не выносить цвет в доп переменную. Чтобы получить нужную прозрачность надо написать так:
```suggestion
  background-color: rgba($basic-green, 0.85);
```